### DISCUSSION TITLE
I want to change the order of the docs sidebar

### DISCUSSION THREAD

**Discussion by Marco5dev** [thuliteio/doks#1098] [Q&A] [answered]
## I want to change the order of the sidebar next to it in the docs, but I cannot

do some one now how to do it ?

**Comment by Marco5dev** [comment]
![image](https://github.com/h-enk/doks/assets/100785809/01d48f03-18f9-4d9d-9b5b-86e716a8b85c)
![image](https://github.com/h-enk/doks/assets/100785809/2bd9ba76-a82f-46b5-a8c7-22aa3887c844)


as you see in the image Get Started is the last one and i want it to be the first one

**Comment by h-enk** [comment, marked as answer]
To change the order, you can use [`weight`](https://gohugo.io/templates/lists/#order-content) in the frontmatter of pages:

- For branch pages, e..g. for `content/docs/getting-started/_index.md` set `weight: 100`. 
- Within a branch, for leaf pages, e.g. for `content/docs/getting-started/installation/index.md` set `weight: 110`

Next branch, use  `weight: 200` and so on
